src/taipy/core/_scheduler/_dispatcher/_development_job_dispatcher.py

```diff
@@ -30,3 +32,26 @@ def is_running(self) -> bool:
 
     def stop(self):
         return NotImplemented
+
+    def run(self):
+        return NotImplemented
+
+    def _dispatch(self, job: Job):
+        """Dispatches the given `Job^` on an available worker for execution.
+
+        Parameters:
+            job (Job^): The job to submit on an executor with an available worker.
+        """
+        self._nb_available_workers -= 1
```

Comment:
I believe _nb_available_workers is not necessary anymore since we are synchronous with a single worker.

Reply:
agree, we can remove it. I keep it there just for the sake of consistency really but like you said, it's not needed at all!